testing/mod.ts

```diff
@@ -125,20 +125,24 @@ async function runTests() {
   for (let i = 0; i < tests.length; i++) {
     const { fn, name } = tests[i];
     let result = green_ok();
-    console.log("test", name);
+    // See https://github.com/denoland/deno/pull/1452
+    // about this usage of groupCollapsed
+    console.groupCollapsed(`test ${name} `);
```

Comment:
I got errors like:
```
/Users/jung/workspace/etc/deno_std/testing/mod.ts:131:13 - error TS2339: Property 'groupCollapsed' does not exist on type 'Console'.

131     console.groupCollapsed(`test ${name} `);
                ~~~~~~~~~~~~~~
/Users/jung/workspace/etc/deno_std/testing/mod.ts:136:15 - error TS2339: Property 'groupEnd' does not exist on type 'Console'.

136       console.groupEnd();
                  ~~~~~~~~
/Users/jung/workspace/etc/deno_std/testing/mod.ts:140:15 - error TS2339: Property 'groupEnd' does not exist on type 'Console'.

140       console.groupEnd();
```
Is it only working on newest 'deno' version?

Thanks.

Reply:
Yes. `console.group*` methods are shipped for the first time in v0.2.6 (released a few days ago). You need v0.2.6 to run this patch.